### PR TITLE
Issue #216 Improve startup message to include selected driver

### LINKED ISSUE #216: Improve startup message to include selected driver
I think we should add the selected driver to the startup message to make it more explicit which driver is selected. See also question #215.

Currently we have:

> Starting local OpenShift cluster...

I suggest we have something like this

> Starting local OpenShift instance using \<driver\> hypervisor ...

### PR DESCRIPTION
**Pull request by budhrg**
Fix #216

### CODE REVIEW

**Comment by LalatenduMohanty** [comment]
Tested on OSX+xhyve and Virtualbox and it works fine.

File: cmd/minikube/cmd/start.go [hunk 1]
```diff
@@ -126,6 +124,7 @@ func runStart(cmd *cobra.Command, args []string) {
 
 	var host *host.Host
 	start := func() (err error) {
+		fmt.Printf("Starting local OpenShift instance using '%s' hypervisor...\n", config.VMDriver)
```
Comment (praveenkumar) [inline cmd/minikube/cmd/start.go]: LGTM :+1: , I am not sure we can add any test case for this change because that's a print statement.

Comment (budhrg) [inline cmd/minikube/cmd/start.go]: Ya, I am not able to think what approach to use to write any test case. Let me know @hferentschik , if you have any approach.

Comment (hferentschik) [inline cmd/minikube/cmd/start.go]: >  I am not able to think what approach to use to write any test case for above change. Let me know @hferentschik , if you have any approach.

This is very much testable :-), however, I think in this case it is not worth the effort. 

One way to test this, is to simply capture the stdout and make sure the log statement comes out. Eventually we will need some helper utility for that. For now we should be good.

Comment (hferentschik) [inline cmd/minikube/cmd/start.go]: @budhrg any reason why you moved the log statement. You put it at a place where it potentially will be printed several times. I am inclined to move it back.

File: README.md [hunk 2]
```diff
@@ -113,7 +113,8 @@ The interaction with OpenShift is via the command line tool _oc_ which is copied
            To login as administrator:
                oc login -u system:admin
 
-    Note that, the IP seen above is dynamic and can change. It can be retrieved with `minishift ip`.
+    Note that, the IP seen above is dynamic and can change. It can be retrieved with `minishift ip`. Also,
+    instead of 'kvm', you will see 'xhyve' in Mac OS and 'hyperv' in Windows.
```
Comment (hferentschik) [inline README.md]: 'on'

File: cmd/minikube/cmd/start.go (hunk 1 above)
Comment (hferentschik) [inline cmd/minikube/cmd/start.go]: I guess you need the config, but you still don't have to put it into the function which gets retried.

**Comment by hferentschik** [comment]
Merged with some minor modifications by adding commit 7c32e0b67cfad65c446c12160a666797f2bc0b4f

**Pull request closed without merging**